src/components/chatroom/DateSeparator.tsx

```diff
@@ -0,0 +1,24 @@
+interface DateSeparatorProps {
+  date: string; // "2025년 9월 9일" 형식
+  isFirstMessage?: boolean; // 첫 메시지인지 여부
+}
+
+const DateSeparator = ({ date, isFirstMessage = false }: DateSeparatorProps) => {
+  return (
+    <div className={`flex justify-center items-center ${isFirstMessage ? 'mb-4' : 'mt-8 mb-4'}`}>
+      <div 
+        className="flex justify-center items-center px-3 py-[6px] rounded-[30px]"
+        style={{
+          background: 'rgba(255, 255, 255, 0.20)',
+          gap: '10px'
+        }}
+      >
```

Comment:
```suggestion
<div className="flex items-center justify-center gap-2.5 px-3 py-1.5 rounded-[30px] bg-white/20">
```
모두 tailwind에서 이미 지원하는 속성들이니 굳이 inline style로 지정하지 않아도 될 것 같아요! 다른 코드에서도 비슷하게 혼용하는 부분이 보이는데, 꼭 필요한 부분이 아니면 통일하는 것이 좋을 것 같습니당 👍